aiqleads/utils/logging.py

```
@@ -2,17 +1,64 @@
+def get_log_path(date: Optional[str] = None) -> str:
+    """Get path to log file for specific date.
     
-    return logger
+    Args:
+        date: Date string in YYYY-MM-DD format (default: today)
+        
+    Returns:
+        str: Path to log file
+    """
+    if date is None:
+        date = datetime.now().strftime("%Y-%m-%d")
```

Ruff (0.8.2)

`datetime.datetime.now()` called without a `tz` argument

(DTZ005)